sdk/cosmosdb/cosmos/test/public/common/TestHelpers.ts

```diff
@@ -77,6 +77,19 @@ export async function getTestContainer(
   return db.container(id);
 }
 
+export async function getmaxIntegratedCacheStalenessTestContainer(
```

Comment:
What's the validation/test goal? 
If its to send header over wire, then can you please share the related test coverage?

Reply:
Waiting on it.

Reply:
This is a "helper" function. The caller was in a previous commit, and that code is no longer needed, but it's good to keep a helper function.

Reply:
What's caller behavior with-out this implementation before?